build.gradle

```diff
@@ -55,9 +55,8 @@ application {
 
 // TODO: Ugly workaround to temporarily ignore build errors to dependencies of latex2unicode
 // These should be removed, as well as the files in the lib folder, as soon as they have valid module names
-modularity.patchModule("test", "fastparse_2.12-1.0.0.jar")
-modularity.patchModule("test2", "fastparse-utils_2.12-1.0.0.jar")
-modularity.patchModule("test3", "sourcecode_2.12-0.1.4.jar")
+modularity.patchModule("fastparse", "sourcecode_2.13-0.2.3")
```

Comment:
Not completely sure, but the module name of sourcecode was fixed in https://github.com/com-lihaoyi/sourcecode/pull/49.

Reply:
Nice! Than you should also be able to delete the files in /lib

Reply:
That was not possible because scala does not yet implement the module system fully and when it is added as dependency, the automatic module naming was getting in the way

Reply:
Strange, the automatic module name should have actually fixed these problems.